decred_wallet/Constants/Constants.swift

```diff
@@ -30,6 +30,7 @@ struct GlobalConstants {
         static let orangeColor = UIColor(hex: "fd714a")
         static let navigationBarColor = UIColor(hex: "689F38")
         static let lightGrey = UIColor(red: 236.0/255.0, green: 238.0/255.0, blue: 241.0/255.0, alpha: 1.0)
+        static let BorderlightGrey = UIColor(red:222/255, green:225/255, blue:227/255, alpha: 1)
```

Comment:
- This should go to `Extensions/UIColor.swift`. See the comment above this `Colors` struct.
- Color names should not include where they're used as though they can't be used elsewhere. Take off the `Border` text from the variable name and describe the color alone. We already have `lightGrey`, so one can be `lighterGrey` and the other `lightGrey`.
- Variable name should start with small letter.
- Use hex code for the color please.